### ISSUE TITLE
C3D pretrained model

### ISSUE THREAD

**Issue by cristinasegalin** [Cadene/pretrained-models.pytorch#33]
Would be possible to have the 3d cnn pretrained model?
Thanks

**Comment by Cadene** [comment]
@cristinasegalin I am sorry but I don't know this network, do you have any idea where I could find the pretrained version of this network?

**Comment by cristinasegalin** [comment]
Hi,
You can find everything related to the model at
http://vlg.cs.dartmouth.edu/c3d/
It's in caffe
Hope this help

Cristina

**Comment by Cadene** [comment]
@cristinasegalin Having a pretrained model for video classification is a good idea. It is not that hard to port this model, however I would need to test it on the same video dataset.

Do you have an implementation with torch.data.Dataset of this dataset?

**Comment by cristinasegalin** [comment]
No, that is actually what I was looking for in your github overall the
available models.


Cristina
